classes/Product.php

```diff
@@ -2226,7 +2238,19 @@ public function updateAttribute(
                 'default_on' => null !== $default,
                 'minimal_quantity' => null !== $minimal_quantity,
                 'available_date' => null !== $available_date,
-            ]);
+            ];
+            if ($available_now !== null) {
+                foreach($available_now as $id_lang => $value) {
+                    $fieldsToUpdate['available_now'][$id_lang] = true;
+                }
+            }
+            if ($available_later !== null) {
```

Comment:
```suggestion
            if (is_array($available_later)) {
```

Reply:
I am not sure about this, it can also be a string and it would be saved fine by the Object model 🤔